http.go

```diff
@@ -25,7 +27,9 @@ func StartStopTLSServer(ctx context.Context, srv *http.Server, shutdownTimeout t
 		c, cancel := context.WithTimeout(context.Background(), shutdownTimeout)
 		defer cancel()
 		if srv.Shutdown(c) != nil {
-			srv.Close()
+			if err := srv.Close(); err != nil {
+				logz.LoggerStr("default", "default").Error("Server listener failed to close", zap.Error(err))
+			}
```

Comment:
Another unhandled error, and another potentially clumsy attempt at handling it

Reply:
This is also not needed - the program is shutting down and if for some reason the http server couldn't cleanly shut down it is not a big deal. Remove please.